RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -100,9 +100,24 @@ boolean pathAllowsFamiliar()
 	return true;
 }
 
+boolean pathAllowsChangingFamiliar()
+{
+    if(!pathHasFamiliar())
+    {
+        return false;
+    }
+
+    if ( $strings[Quantum Terrarium] contains auto_my_path() )
```

Comment:
needs a comment explaining that this path has a familiar but forces you to use one of its choice

Reply:
Comment added in new version